clang/include/clang/Driver/Options.td

```diff
@@ -6911,6 +6911,10 @@ def fsycl_fp32_prec_sqrt : Flag<["-"], "fsycl-fp32-prec-sqrt">, Group<sycl_Group
   Visibility<[ClangOption, CC1Option]>,
   HelpText<"SYCL only. Specify that single precision floating-point sqrt is correctly rounded.">,
   MarshallingInfoFlag<CodeGenOpts<"SYCLFp32PrecSqrt">>;
+def fsycl_use_builtins_for_integration : Flag<["-"], "fsycl-use-builtins-for-integration">,
+  Group<sycl_Group>, Visibility<[ClangOption, CLOption, CC1Option]>,
+  HelpText<"SYCL only. Specify that builtin functions be used for integrating the host and the device compilations.">,
```

Comment:
Since the integration is unidirectional, I think we can simplify the help text.
```suggestion
  HelpText<"SYCL only. Specify that kernel information is exposed to the SYCL host run-time library via builtin functions.">,
```

Reply:
I changed the wording of the help text, but not quite to what you suggested.   Please review.